ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -279,3 +279,52 @@ periodics:
         requests:
           cpu: 10m
     serviceAccountName: ci-operator
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: openshift-azure-not-running-under-test
```

Comment:
lets keep azure in the front for all jobs, helps to filter easier in prow `azure-test-not-rut`?

Reply:
makes sense, will update.